content/07.pathogenesis.md

```diff
@@ -21,12 +22,26 @@ The C-terminal domain of the SARS-CoV-2 S protein in particular was identified a
 Recent crystal structures have shown that the receptor-binding domain (RBD) of SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507]. 
 Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach. Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that from SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind SARS-CoV-2-RBD [@doi:10.1126/science.abb2507]. 
-Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052]. 
+Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052].
 
 #### Viral Replication and Spreading
 
--Basic introduction into replication cycle
--What are the routes of transmission
+- Basic introduction into replication cycle
+- What are the routes of transmission
+
+##### Mechanism of viral replication within cells
+
+##### Mechanism of viral spreading to neighbor cells
+
+##### Factors enhancing viral spreading
+
+Viral progression may be enhanced by active upregulation of ACE2 on cell surfaces following or during a response to infection.
+In several preliminary assays and an analysis of previous microarray data, Wang et. al. reported that ACE2 is significantly upregulated following infection of other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus [@doi:10.1101/2020.02.24.963348].
+Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression as compared to control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
+Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies. 
+One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, and may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
```

Comment:
We also have some elements on the cytokine storm in the Tocilizumab section: https://greenelab.github.io/covid19-review/#tocilizumab

I have created an issue to track a potential solution (filling in 2.2): https://github.com/greenelab/covid19-review/issues/233